spec/components/tailwinds/form/builder_spec.rb

```diff
@@ -7,13 +7,32 @@
   let(:builder) { Tailwinds::Form::Builder.new :user, resource, view, {} }
 
   describe '#text_field' do
-    let(:output) do
-      builder.text_field :email
+    context 'default' do
+      let(:output) { builder.text_field :email }
+
+      it do
+        expect(output).to have_selector 'label.block.text-gray-700.text-sm.font-bold.mb-2'
+        expect(output).to have_selector 'input.w-full.px-3.py-2.border.border-gray-300.rounded'
+      end
     end
 
-    it do
-      expect(output).to have_selector 'label.block.text-gray-700.text-sm.font-bold.mb-2'
-      expect(output).to have_selector 'input.w-full.px-3.py-2.border.border-gray-300.rounded'
+    context 'with value from options' do
+      let(:value) { 'leopold@purple-magic.com' }
+      let(:output) { builder.text_field :email, value: }
+
+      it 'gets value from options' do
+        expect(output).to have_selector "input[value='#{value}']"
+      end
+    end
+
+    context 'with value from object' do
+      before { resource.email = email }
```

Comment:
i think its better to add empty line after line 29

Reply:
done!